docs/resources/frontend_o11y_app.md

```diff
@@ -0,0 +1,58 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "grafana_frontend_o11y_app Resource - terraform-provider-grafana"
+subcategory: "Frontend Observability"
+description: |-
+  
+---
+
+# grafana_frontend_o11y_app (Resource)
+
+
+
+## Example Usage
+
+```terraform
+data "grafana_cloud_stack" "teststack" {
+  provider    = grafana.cloud
+  name        = "gcloudstacktest"
+}
+
+resource "grafana_frontend_o11y_app" "test-app" {
```

Comment:
Is it possible to set `Combine K6 with RUM data` in this resource also for each frontend o11y app?

Reply:
Yes! Its possible from the settings using the "combineLabData" setting